modules/microshift-manifests-deletion-overview.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-manifests-deletion-overview_{context}"]
+= How manifest deletion works
+
+By default, {microshift-short} expects to find deletion manifests in the `delete` subdirectories within the manifest path. When a manifest file is placed in these subdirectories, {microshift-short} automatically handles the removal. Read through the following procedure to understand how manifest file deletion works in {microshift-short}.
```

Comment:
It handles deletion automatically, but only during start

```suggestion
By default, {microshift-short} expects to find deletion manifests in the `delete` subdirectories within the manifest path. When a manifest file is placed in these subdirectories, {microshift-short} automatically handles the removal during start. Read through the following procedure to understand how manifest deletion works in {microshift-short}.
```

Reply:
done